NAMESPACE

```diff
@@ -17,8 +17,11 @@ export(import_vega_data)
 export(install_altair)
 export(knit_print.altair.vegalite.v2.api.TopLevelMixin)
 export(knit_print.vegaspec)
+export(py_mimebundle)
 export(r_to_py)
 export(renderVegawidget)
+export(repr_text.altair.vegalite.v2.api.Chart)
+export(repr_vegalite2.altair.vegalite.v2.api.Chart)
```

Comment:
So here’s the thing: `roxygen2` tries to be smart with S3 exports. Once my repr PR has landed and repr is loaded while recreating the NAMESPACE file with roxygen, roxygen will put `S3Method(repr_vegalite2, altair.vegalite.v2.api.Chart)` here. You can control this by manually doing

- `#' @export repr_vegalite2.altair.vegalite.v2.api.Chart` to force exporting it with its full name, or
- `#' @importFrom repr repr_vegalite2` to force exporting it as a S3 method (once my PR has landed)

both is valid, but I think people prefer the latter.

Reply:
I will go with the latter, can you give me a heads-up when it will have landed?

Reply:
I see it repr 0.18 is on CRAN (congrats!) - I will proceed.